WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipment Tracking Section/Add Tracking/ManualTrackingViewController.swift

```diff
@@ -134,7 +136,13 @@ private extension ManualTrackingViewController {
     }
 
     @objc func dismissButtonTapped() {
-        dismiss()
+        if viewModel.canCommit || viewModel.hasUnsavedChanges {
+            displayDismissConfirmationAlert()
+        } else if viewModel.isEmptyState {
```

Comment:
It is not necessary to add the `else if viewModel.isEmptyState {` here, with just 
```
} else {
            dismiss()
        }
```
it is enough.

Reply:
Thanks, I was wondering about this one in case there were extra unaccounted cases, but I see all would fall into the else statement anyway 👍